src/main/java/core/lib/PasswordValidator.java

```diff
@@ -0,0 +1,30 @@
+package core.lib;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class PasswordValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String field;
+    private String fieldMatch;
+    
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        this.field = constraintAnnotation.field();
+        this.fieldMatch = constraintAnnotation.fieldMatch();
+    }
+    
+    @Override
+    public boolean isValid(Object obj, ConstraintValidatorContext cvc) {
+        Object fieldValue = new BeanWrapperImpl(obj)
+                .getPropertyValue(field);
+        Object fieldMatchValue = new BeanWrapperImpl(obj)
```

Comment:
```suggestion
    public boolean isValid(UserRequestDto userRequestDto, ConstraintValidatorContext cvc) {
        Object fieldValue = new BeanWrapperImpl(userRequestDto)
                .getPropertyValue(field);
        Object fieldMatchValue = new BeanWrapperImpl(userRequestDto)
```

Reply:
If I don't change the generic type. There is no need to make these changes